app/models/comment.rb

```diff
@@ -0,0 +1,7 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+  validates :user, presence: true
+  validates :post, presence: true
```

Comment:
There is no need to validate presence of user and post for this model. The belongs_to association will enforce the validation.